source/install/self-managed-changelog.md

```diff
@@ -5,12 +5,124 @@
-- [Release v6.4 - Feature Release](#release-v6-4-feature-release)
 - [Release v6.3 - Extended Support Release](#release-v6-3-extended-support-release)
 
+## Release v7.0 - [Major Release](https://docs.mattermost.com/upgrade/release-definitions.html#major-release)
+
+**Release Day: 2022-06-15**
+
+### Important Upgrade Notes
+ - MySQL self-hosted customers may notice the migration taking longer than usual when having a large number of rows in the ``FileInfo`` table. For MySQL, it takes around 19 seconds for a table of size 0.7M rows. The time required for PostgreSQL is negligible.
+ - When a new configuration setting ``EnableAppBar`` is enabled, all channel header icons registered by plugins will be moved to the new App Bar, even if they do not explicitly use the new registry function to render a component there. The setting ``EnableAppBar`` defaults to ``false`` for self-hosted deployments.
+ - The value of ``ServiceSettings.TrustedProxyIPHeader`` defaults to empty from now on. A previous bug prevented this from happening in certain conditions. Customers are requested to check for these values in their config and set them to nil if necessary.
+ - **IMPORTANT:** Session length configuration settings have changed from days to hours. If anything goes awry during a server upgrade, it could result in users not being able to log in. For customers who use environment variables for these config settings, the automatic migration won't work, and new configuration  environment variables must be added for: ``MM_SERVICESETTINGS_SESSIONLENGTHWEBINHOURS=``, ``MM_SERVICESETTINGS_SESSIONLENGTHMOBILEINHOURS=``, and ``MM_SERVICESETTINGS_SESSIONLENGTHSSOINHOURS=``. The values need to be 24x the existing values in days. For example, if ``MM_SERVICESETTINGS_SESSIONLENGTHWEBINDAYS=30``, they should set ``MM_SERVICESETTINGS_SESSIONLENGTHWEBINHOURS=720``.
+
+**IMPORTANT:** If you upgrade from a release earlier than v6.7, please read the other [Important Upgrade Notes](https://docs.mattermost.com/upgrade/important-upgrade-notes.html).
+
+### Highlights
+
+#### Collapsed Reply Threads (General Availability)
+ - [Collapsed Reply Threads](XXXXXX link to CRT GA blog post) is now generally available. Please reference [this article](link to KB article XXXXXX) for more information and guidance for enabling the feature.
+
+#### Calls (Beta)
+ - Native voice calling and screen sharing is now available. This is a Channels-specific integration.
+
+#### Apps Bar (Beta)
+ - When a new configuration setting ``EnableAppBar`` is enabled, the channel header will be decluttered to make it more obvious how to access Calls, Playbooks, and Boards when viewing a channel. All channel header icons registered by plugins will be moved to the new Apps Bar when the configuration setting is enabled. We recommend enabling the Apps Bar for servers with Calls enabled since the Apps Bar is the dedicated place for the Calls start/join button.
```

Comment:
Propose starting with the benefit, followed by the technical details. E.g. `Channel header is now decluttered ... when a new setting ....`

Reply:
> We recommend enabling the Apps Bar for servers with Calls enabled since the Apps Bar is the dedicated place for the Calls start/join button.

I thought the Calls start/join button stays in the channel header and doesn't move to the Apps Bar?

Reply:
```suggestion
 - When a new configuration setting ``ExperimentalSettings .EnableAppBar`` is enabled, the channel header will be decluttered to make it more obvious how to access Calls, Playbooks, and Boards when viewing a channel. All channel header icons registered by plugins will be moved to the new Apps Bar when the configuration setting is enabled. We recommend enabling the Apps Bar for servers with Calls enabled since the Apps Bar is the dedicated place for the Calls start/join button.
```

Reply:
"I thought the Calls start/join button stays in the channel header and doesn't move to the Apps Bar?" @streamer45 can confirm

Reply:
That's correct. The Apps Bar helps in making space for the dedicated button as any other plugin will go under it instead of filling the channel header.